english/coffee/coffee.md

```diff
@@ -14,6 +14,8 @@ After having your coffee do you want to:
 
 [Remenisce?](../remenisce/better-times.md).
 
+[Bake yourself something sweet to go with this delicious brew?](../cookie/bake-cookies.md)
```

Comment:
I believe you don't need the `../` at the beginning, since the `cookie` directory is in the same directory as the `coffee.md` file.